tests/translator/output/aws-us-gov/function_with_amq.json

```diff
@@ -35,7 +37,9 @@
         "SourceAccessConfigurations": [
           {
             "Type": "BASIC_AUTH",
-            "URI": "arn:aws:secretsmanager:us-west-2:123456789012:secret:my-path/my-secret-name-1a2b3c"
+            "URI": {
+              "Fn::Sub": "arn:${AWS::Partition}:secretsmanager:us-west-2:123456789012:secret:my-path/my-secret-name-1a2b3c"
+            }
           }
```

Comment:
Output templates should not change so we can ensure backward compatibility. Please add new transform tests instead.

Reply:
I ran templates with Function + MQ with `bin/add_transform_test.py` and had the error like “hardcoded partition is detected, you shouldn’t do that” I suppose `add_transform_test` changed over time. So instead of `arn:aws:` in all urls I changed it to `!Sub 'arn:${AWS::Partition}:`. It's the only change to the existing transformation tests. 

Plus, I've added one new test for multiple MQ event sources.

Reply:
We should ensure existing templates that deploy successfully transform exactly the same, and either:

1. Add a property to opt-in to the new behavior (e.g. `DynamicPolicyName: true`... naming is hard).
2. Figure out a way to do it in a backward-compatible manner without new properties (might be tricky, need to be mindful of reading order, etc.).